accounts/management/commands/add_staff_users.py

```diff
@@ -0,0 +1,71 @@
+import csv
+from django.core.management.base import BaseCommand, CommandError
+from noiseworks import cobrand
+from accounts.models import User
+
+
+def ward_name_to_id(ward):
+    wards = cobrand.api.wards()
+    wards = {ward["name"]: ward["gss"] for ward in wards}
+    try:
+        return wards[ward]
+    except:
+        raise CommandError(f"Could not find ward {ward}")
+
+
+def obj_or_string_dictreader(filename):
```

Comment:
I don't quite understand what this is for, but I'm guessing it's so the tests can pass `StringIO` objects in? A little explanatory comment would be good. Alternatively your tests could pass [a list of strings](https://docs.python.org/3/library/csv.html#csv.reader) instead of `StringIO` and this function wouldn't be needed at all, I think.

Reply:
If you call it at the command line with a filename, it needs to call `open()` on it, if it comes from the test (whether it's StringIO or a list) it won't need to. I guess best solution would be if the test could pass in something on which open() can be called, I'll have a think.

Reply:
Any reason the test CSVs couldn't be real files on disk?

Reply:
They're just so small, find it easier if the data is in the test. I think I've done it - management command doesn't have anything special, just uses `open()` as normal, test has a monkeypatch on open that returns fixed data depending on the filename.